content/en/docs/spin-operator/tutorials/pub-sub-with-redis.md

```diff
@@ -0,0 +1,219 @@
+---
+title: Publish-Subscribe With Redis
+description: Learn how to create a Spin App that responds to messages on pub-sub Redis channels and runs in Kubernetes
```

Comment:
We should update this to use [Valkey](https://github.com/valkey-io/valkey) rather than Redis now that Redis is no-longer open source software.